main.rb

```diff
@@ -0,0 +1,70 @@
+require './app' # import the App class
+
+def main # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/CyclomaticComplexity
+  app = App.new
```

Comment:
### OPTIONAL

```suggestion
  app = App.new()
```

- Always add the `()` after a method, this is the conventional way of writing methods